src/lib/app/wait.tsx

```diff
@@ -29,3 +29,29 @@ export async function waitUntilAppIsInstalled(
 
   stepWaiting.complete();
 }
+
+export async function waitUntilAppIsUpgraded(
```

Comment:
Apart from the label text, this function is identical to `waitUntilAppIsInstalled`. This is a lot of duplication and should be simplified.

Reply:
Oh yeah, you're completely right.
I propose using one "waitUntilAppStateHasNormalized"-function with one additional parameter wich is the label.
ab1ab4b251d7b27c441b34889709bbc39299a3f6